### PR TITLE
Create UI Play Music

### PR DESCRIPTION
**Pull request by VanHieu7119**
Create UI play music 
Link Redmine: https://edu-redmine.framgia.vn/issues/2701
image

### CODE REVIEW

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        android:layout_height="@dimen/dp_30"
+        android:layout_marginLeft="@dimen/dp_20"
+        android:ellipsize="end"
+        android:maxLines="1"
+        android:textColor="@color/colorTextView"
+        android:textSize="@dimen/sp_15"
+        app:layout_constraintLeft_toRightOf="@id/buttonBack"
+        app:layout_constraintTop_toBottomOf="@+id/textViewNameSong"
+
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank (51-52)

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        android:id="@+id/buttonDownload"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/dp_15"
+        android:background="@android:color/transparent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_download_black_36dp"
+        tools:ignore="VectorDrawableCompat"
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        android:src="@drawable/ic_disc1"
+        app:civ_border_color="#101529"
+        app:civ_border_width="@dimen/dp_2"
+        app:layout_constraintBottom_toTopOf="@id/seekBar"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textViewNameArtist"
+        app:layout_constraintVertical_bias="0.3"
+        />
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank(79-80)

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/dp_10"
+        android:background="@android:color/transparent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toRightOf="@+id/buttonShuffle"
+        app:layout_constraintRight_toLeftOf="@+id/buttonPlay"
+        app:srcCompat="@drawable/ic_previous_black_24dp"
+        tools:ignore="VectorDrawableCompat"
+        />
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    <ImageButton
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/dp_10"
+        android:background="@android:color/transparent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toRightOf="@+id/buttonPlay"
+        app:layout_constraintRight_toLeftOf="@id/buttonRepeat"
+        app:srcCompat="@drawable/ic_next_black_24dp"
+        tools:ignore="VectorDrawableCompat"
+        />
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/dp_30"
+        android:background="@android:color/transparent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toRightOf="@id/buttonNext"
+        app:layout_constraintRight_toRightOf="parent"
+        app:srcCompat="@drawable/ic_repeat_black_24dp"
+        tools:ignore="VectorDrawableCompat"
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: remove blank

**Comment by tuanlvt-1117** [comment]
commented

**Comment by VanHieu7119** [comment]
ready

File: app/src/main/res/layout/fragment_content_play_music.xml [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_download_black_36dp"
+        tools:ignore="VectorDrawableCompat"
+        />
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/imageCD"
+        android:layout_width="@dimen/dp_250"
+        android:layout_height="@dimen/dp_250"
+        android:src="@drawable/ic_disc1"
+        app:civ_border_color="#101529"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_content_play_music.xml]: hard code

**Comment by tuanlvt-1117** [comment]
commented

**Comment by VanHieu7119** [comment]
ready

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**